### PR TITLE
feat(mizrahi-scraper): add update email page support

### PR DESCRIPTION
**Pull request by naorpeled**

### CODE REVIEW

File: src/scrapers/mizrahi.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ function createLoginFields(credentials: ScraperCredentials) {
 
 function getPossibleLoginResults(page: Page): PossibleLoginResults {
   return {
-    [LoginResults.Success]: [AFTER_LOGIN_BASE_URL],
+    [LoginResults.Success]: [AFTER_LOGIN_BASE_URL, async () => !!(await page.$x('//title[contains(., "עדכון כתובת דוא\'\'ל")]'))],
```
Comment (naorpeled) [inline src/scrapers/mizrahi.ts:69]: Would add an XPath expression generation helper to increase readability and ease of use for cases when there are several items within a contains clause for example.
Not in the scope of this PR though.

File: src/scrapers/mizrahi.ts [hunk 2]
```diff
@@ -66,7 +69,7 @@ function createLoginFields(credentials: ScraperCredentials) {
 
 function getPossibleLoginResults(page: Page): PossibleLoginResults {
   return {
-    [LoginResults.Success]: [AFTER_LOGIN_BASE_URL],
+    [LoginResults.Success]: [AFTER_LOGIN_BASE_URL, async () => !!(await page.$x(`//title[contains(., "${emailUpdatePageHebrewTitle}") or contains(., "${emailUpdatePageEnglishTitle}")]`))],
```
Comment (naorpeled) [inline src/scrapers/mizrahi.ts:72]: Would add an XPath expression generation helper to increase readability and ease of use for cases when there are several items within a contains clause for example.
Not in the scope of this PR though.

Comment (esakal) [reply src/scrapers/mizrahi.ts:72]: Hi, thanks for the contribution. Just to verify that I understand the reason behind your change. From the code you have scenario that you are not navigated to `AFTER_LOGIN_BASE_URL` and instead you reach an other page (or maybe the same one as the login) which should the update email title. And although you see that popup, you are already logged so you can proceed with the scrapping?

Comment (naorpeled) [reply src/scrapers/mizrahi.ts:72]: Yes,
it's a page within the dashboard,
you can proceed with doing any actions without updating your email.

**Pull request merged**